packages/cli/src/utils/miscellaneous.ts

```diff
@@ -356,33 +364,20 @@ export function printConfigLintTotals(totals: Totals, command?: string | number)
   }
 }
 
-export function getOutputFileName(
-  entrypoint: string,
-  entries: number,
-  output?: string,
-  ext?: BundleOutputFormat
-) {
+export function getOutputFileName(entrypoint: string, output?: string, ext?: BundleOutputFormat) {
   if (!output) {
-    return { outputFile: 'stdout', ext: ext || 'yaml' };
+    return { ext: ext || 'yaml' };
   }
 
   let outputFile = output;
-  if (entries > 1) {
-    ext = ext || (extname(entrypoint).substring(1) as BundleOutputFormat);
-    if (!outputExtensions.includes(ext as any)) {
-      throw new Error(`Invalid file extension: ${ext}.`);
-    }
-    outputFile = join(output, basename(entrypoint, extname(entrypoint))) + '.' + ext;
-  } else {
-    if (output) {
-      ext = ext || (extname(output).substring(1) as BundleOutputFormat);
-    }
-    ext = ext || (extname(entrypoint).substring(1) as BundleOutputFormat);
-    if (!outputExtensions.includes(ext as any)) {
-      throw new Error(`Invalid file extension: ${ext}.`);
-    }
-    outputFile = join(dirname(outputFile), basename(outputFile, extname(outputFile))) + '.' + ext;
+  if (output) {
+    ext = ext || (extname(output).substring(1) as BundleOutputFormat);
+  }
+  ext = ext || (extname(entrypoint).substring(1) as BundleOutputFormat);
+  if (!outputExtensions.includes(ext)) {
+    throw new Error(`Invalid file extension: ${ext}.`);
   }
+  outputFile = join(dirname(outputFile), basename(outputFile, extname(outputFile))) + '.' + ext;
```

Comment:
I think there may be aproblem here. The `output` in `redocly.yaml` should be relative to the `reodcly.yaml`'s folder, not to the current working dir.

Reply:
You're right, good catch!